app/src/store/playgroundStore.tsx

```diff
@@ -5,7 +5,8 @@ export type GenAIOperationType = "chat" | "text_completion";
 
 let playgroundInstanceIdIndex = 0;
 let playgroundRunIdIndex = 0;
-let playgroundMessageIdIndex = 0;
+// This value must be truthy in order for message re-ordering to work
+let playgroundMessageIdIndex = 1;
```

Comment:
didn't bottom this out but an item with id of 0 is not draggable, assume there is some if(id) check going on somewhere

Reply:
oh insteresting